docs/cloud-workload-security/backend.md

```diff
@@ -18,6 +18,20 @@ CWS logs have the following JSON schema:
 {
     "$id": "https://github.com/DataDog/datadog-agent/pkg/security/serializers/event",
     "$defs": {
+        "AnomalyDetectionSyscallEvent": {
+            "properties": {
+                "syscall": {
+                    "type": "string",
+                    "description": "Syscall is the name of the syscall that triggered the anomaly detection"
```

Comment:
Hey May 👋🏻 We already have this pattern for the entire documentation, and unfortunately we generate the documentation automatically from our comments in code (which have to have the name of the structure first because it's a linter requirement for the entire agent).
If you want us to change this description pattern, we need to also change our generation script. Would it be OK for you if we do it in a separate PR ? This one is already huge ...

Reply:
Hey @Gui774ume, ah okay, thanks for explaining. Sure a separate PR make sense. Thanks!